src/lv_gpu/lv_gpu_stm32_dma2d.c

```diff
@@ -186,7 +186,7 @@ void lv_gpu_stm32_dma2d_copy(lv_color_t * buf, lv_coord_t buf_w, const lv_color_
 void lv_gpu_stm32_dma2d_blend(lv_color_t * buf, lv_coord_t buf_w, const lv_color_t * map, lv_opa_t opa,
                               lv_coord_t map_w, lv_coord_t copy_w, lv_coord_t copy_h)
 {
-    invalidate_cache();
+    lv_disp_invalidate_cache(NULL);
```

Comment:
It should invalidate on the being refreshed display, not the default. I suggest
```c
lv_disp_invalidate_cache(_lv_refr_get_disp_refreshing);
```

Reply:
Fixed in my latest commit.